com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -61,6 +61,13 @@ public RpcCommunicator(CommunicatorInitParameters communicatorInitParameters)
 
         #region Initialization
 
+        internal static bool CheckCommunicationVersionsAreCompatible(string unityCommunicationVersion, string pythonApiVersion)
+        {
+            var unityVersion = new Version(unityCommunicationVersion);
+            var pythonVersion = new Version(pythonApiVersion);
+            return unityVersion.Major == pythonVersion.Major;
```

Comment:
Depending on whether you want to wait for 1.0.0, we should check minor versions if the major is 0.

Reply:
and log a warning?

Reply:
Something like
```csharp
if(unityVersion.Major == 0) {
  return unityVersion.Major == pythonVersion.Major && unityVersion.Minor == pythonVersion.Minor;
} else {
  return unityVersion.Major == pythonVersion.Major;
}
```
?

Reply:
yea, that makes sense